test/EventSourceSpec.js

```diff
@@ -0,0 +1,125 @@
+define(function(require) {
+    'use strict';
+
+    var EventSource = require('wf-js-common/EventSource');
+
+    var testEvent;
+
+    // Phantom doesn't seem to have the support for creating events, so we're relying on saucelabs
+    // to run these tests.  For local development, hit the jasmine test running with the browser
+    // you want to use.
+
+    var canCreateEvent = true;
+    try {
+        testEvent = document.createEvent('Event');
```

Comment:
document.createEvent is deprecated, but some browsers, like Phantom and IE :rotating_light:, do not support newable events yet.